{
  "textPayload": "Traceback (most recent call last):\n  File \"/layers/google.python.pip/pip/lib/python3.9/site-packages/uvicorn/protocols/http/httptools_impl.py\", line 419, in run_asgi\n    result = await app(  # type: ignore[func-returns-value]\n  File \"/layers/google.python.pip/pip/lib/python3.9/site-packages/uvicorn/middleware/proxy_headers.py\", line 78, in __call__\n    return await self.app(scope, receive, send)\n  File \"/layers/google.python.pip/pip/lib/python3.9/site-packages/fastapi/applications.py\", line 271, in __call__\n    await super().__call__(scope, receive, send)\n  File \"/layers/google.python.pip/pip/lib/python3.9/site-packages/starlette/applications.py\", line 118, in __call__\n    await self.middleware_stack(scope, receive, send)\n  File \"/layers/google.python.pip/pip/lib/python3.9/site-packages/starlette/middleware/errors.py\", line 184, in __call__\n    raise exc\n  File \"/layers/google.python.pip/pip/lib/python3.9/site-packages/starlette/middleware/errors.py\", line 162, in __call__\n    await self.app(scope, receive, _send)\n  File \"/layers/google.python.pip/pip/lib/python3.9/site-packages/starlette/middleware/cors.py\", line 92, in __call__\n    await self.simple_response(scope, receive, send, request_headers=headers)\n  File \"/layers/google.python.pip/pip/lib/python3.9/site-packages/starlette/middleware/cors.py\", line 147, in simple_response\n    await self.app(scope, receive, send)\n  File \"/layers/google.python.pip/pip/lib/python3.9/site-packages/starlette/middleware/exceptions.py\", line 79, in __call__\n    raise exc\n  File \"/layers/google.python.pip/pip/lib/python3.9/site-packages/starlette/middleware/exceptions.py\", line 68, in __call__\n    await self.app(scope, receive, sender)\n  File \"/layers/google.python.pip/pip/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py\", line 21, in __call__\n    raise e\n  File \"/layers/google.python.pip/pip/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py\", line 18, in __call__\n    await self.app(scope, receive, send)\n  File \"/layers/google.python.pip/pip/lib/python3.9/site-packages/starlette/routing.py\", line 706, in __call__\n    await route.handle(scope, receive, send)\n  File \"/layers/google.python.pip/pip/lib/python3.9/site-packages/starlette/routing.py\", line 276, in handle\n    await self.app(scope, receive, send)\n  File \"/layers/google.python.pip/pip/lib/python3.9/site-packages/starlette/routing.py\", line 66, in app\n    response = await func(request)\n  File \"/layers/google.python.pip/pip/lib/python3.9/site-packages/fastapi/routing.py\", line 237, in app\n    raw_response = await run_endpoint_function(\n  File \"/layers/google.python.pip/pip/lib/python3.9/site-packages/fastapi/routing.py\", line 163, in run_endpoint_function\n    return await dependant.call(**values)\n  File \"/workspace/app/routers/test_result.py\", line 40, in post_test_result\n    if schema[data_element] != data_unit:\nKeyError: 'srgon'",
  "insertId": "64feb6c7000b7d768ea6ffc2",
  "resource": {
    "type": "gae_app",
    "labels": {
      "project_id": "mobile-doc-backend",
      "version_id": "staging",
      "module_id": "default",
      "zone": "asia-south1-4"
    }
  },
  "timestamp": "2023-09-11T06:42:15.753014Z",
  "severity": "ERROR",
  "labels": {
    "clone_id": "00c61b117c3915c971e9190b9fc8391f23b2a6e3b8db4d93aa46d98c45374cfb970c5ae9a69168f1b1e9ffe4543d15f63c800ebd630e5de047a5"
  },
  "logName": "projects/mobile-doc-backend/logs/stderr",
  "receiveTimestamp": "2023-09-11T06:42:16.053244336Z"
}